frontend/packages/pipelines-plugin/src/components/pipelines/utils/pipeline-operator.ts

```diff
@@ -1,12 +1,22 @@
 import * as React from 'react';
-import { compare, gte, parse, SemVer } from 'semver';
-import { k8sList } from '@console/internal/module/k8s';
+import { compare, gt, gte, parse, SemVer } from 'semver';
+import { useAccessReview } from '@console/dynamic-plugin-sdk';
+import { k8sList } from '@console/dynamic-plugin-sdk/src/utils/k8s';
```

Comment:
can we use `k8sListResource` over `k8sList` ?

Reply:
Keeping it as is as we need to backport it to 4.9 and we don't have `k8sListResource` in previous versions.